Userland/Libraries/LibWeb/HTML/VideoTrackList.cpp

```diff
@@ -127,4 +127,11 @@ WebIDL::CallbackType* VideoTrackList::onremovetrack()
     return event_handler_attribute(HTML::EventNames::removetrack);
 }
 
+void VideoTrackList::visit_edges(JS::Cell::Visitor& visitor)
+{
+    Base::visit_edges(visitor);
+    for (auto const& track : m_video_tracks)
```

Comment:
These are in a `MarkedVector` (same in `AudioTrackList`), so they're safe from GC. If we're visiting them, let's turn that into a plain `Vector`?

Reply:
That's a great point, I missed that. This is a false-positive actually then (though it is a good idea to change this anyways, due to the slight performance penalty for the added root imo)
I'll change these into a plain Vector